octopoes/tools/xtdb_client.py

```diff
@@ -0,0 +1,154 @@
+import datetime
+
+import httpx
+from pydantic import JsonValue
+
+
+class XTDBClient:
```

Comment:
Can't we reuse the client we have in `octopoes/xtdb`? I think that would make sense at least, even if it triggers some refactoring on the module side. Perhaps there's some inspiration to be found in https://github.com/Donnype/xtdb-py/blob/main/xtdb/session.py as well, but perhaps we are tied to some Octopoes specific stuff.